src/trezor-keyring.ts

```diff
@@ -90,7 +151,7 @@ class TrezorKeyring extends EventEmitter {
     TrezorConnect.dispose();
```

Comment:
Potentially out of scope for this PR, but `TrezorConnect.dispose()` returns a promise, so making this promise access to outside the class might be handy. E.g.:
 
```ts
   dispose(): Promise<void> {
       return TrezorConnect.dipose():
    }
```